packages/block-library/src/page-list-item/style.scss

```diff
@@ -0,0 +1,17 @@
+// The Page List block should inherit navigation styles when nested within it
```

Comment:
Same question here about doubling up the styles.

Reply:
I think it's safe to delete the ones in the new block.